client/src/index.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import 'font-awesome/css/font-awesome.min.css';
```

Comment:
I don't think we're using font-awesome yet... Please let me know if I'm missing something, otherwise please remove it unless we're using it.